changelog/2.099.1_pre.dd

```diff
@@ -0,0 +1,144 @@
+    }
+
+    foreach(idx, elem; top)
+    {
+        // Previously:
+        // static assert(is(typeof(__traits(parameters)) == AliasSeq!(size_t, Tree)));
+
+        // Now:
+        static assert(is(typeof(__traits(parameters)) == AliasSeq!(Tree, int)));
+    }
+}
+---
+)
+
+
+)
+
+$(BUGSTITLE_TEXT_BODY Library changes,
+
+$(LI $(LNAME2 log_default_info,std.experimental.logger default level changed to `info` instead of `warning`)
+$(CHANGELOG_SOURCE_FILE phobos, changelog/log_default_info.dd)
+$(P
+In the 2.099.0 release, the default log level $(LINK2 https://dlang.org/changelog/2.099.0.html#logger_default_warning, was changed to warning).
+)
+
+$(P
+However, this was erroneously assumed to include `info` (informational) logs as well. Now it does include `info` log messages.
+)
+)
+
+
+)
+
+$(CHANGELOG_SEP_TEXT_BUGZILLA)
+
+$(BUGSTITLE_BUGZILLA DMD Compiler regression fixes,
+
+$(LI $(BUGZILLA 20015): [REG 2.086] Deprecated -preview, -revert, and -transition options not documented)
+$(LI $(BUGZILLA 20717): Unsilenced bogus "undefined identifier" error from speculative collision)
+$(LI $(BUGZILLA 21285): Delegate covariance broken between 2.092 and 2.094 $(LPAREN)git master$(RPAREN).)
+$(LI $(BUGZILLA 22175): assert fail when struct assignment value is desired and struct size is odd)
+$(LI $(BUGZILLA 22639): Copy constructors with default arguments not getting called)
+$(LI $(BUGZILLA 22858): [REG2.099] Incorrect alignment of void*[0])
+$(LI $(BUGZILLA 22859): Error: forward reference of variable `isAssignable` for mutually recursed `allSatisfy`)
+$(LI $(BUGZILLA 22860): `Error: unknown` with mutually recursive and nested SumType)
+$(LI $(BUGZILLA 22863): [REG2.099] -main doesn't work anymore when used for linking only $(LPAREN)without source modules$(RPAREN))
+)
+$(BUGSTITLE_BUGZILLA DMD Compiler bug fixes,
+
+$(LI $(BUGZILLA 22852): importC: Lexer allows invalid wysiwyg and hex strings)
+$(LI $(BUGZILLA 22853): importC: Lexer allows nesting block comments)
+$(LI $(BUGZILLA 22868): __traits$(LPAREN)parameters$(RPAREN) returns parameters of delegate instead of function)
+$(LI $(BUGZILLA 22871): Using an alias to `__traits$(LPAREN)parameters$(RPAREN)` causes unknown error)
+$(LI $(BUGZILLA 22939): bad error message: Error: no property `msg` for type `string`)
+$(LI $(BUGZILLA 22942): Invalid section type / offset for newer XCode versions)
+)
+$(BUGSTITLE_BUGZILLA Druntime regression fixes,
+
+$(LI $(BUGZILLA 20778): exception messages with nulls within are treated inconsistently)
+)
+$(BUGSTITLE_BUGZILLA Druntime enhancements,
+
+$(LI $(BUGZILLA 22908): OpenBSD: Add getpwnam_shadow and getpwuid_shadow function prototypes)
+)
+$(BUGSTITLE_BUGZILLA dlang.org bug fixes,
+
+$(LI $(BUGZILLA 21086): Wrong source link for core.thread.context)
+)
+)
+$(D_CONTRIBUTORS_HEADER 11)
+$(D_CONTRIBUTORS
+    $(D_CONTRIBUTOR Brian Callahan)
+    $(D_CONTRIBUTOR Dennis)
+    $(D_CONTRIBUTOR Dennis Korpel)
+    $(D_CONTRIBUTOR dkorpel)
```

Comment:
I wonder why I'm listed three times, where does it get the names from?

Reply:
There is a list somewhere that deduplicated based on aliases. 
I would guess one is your GitHub name, one is the email used to sign your git commits, don't know about the third.

Reply:
Uses [`.mailmap`](https://github.com/dlang/tools/blob/608a5d07705f244c537741c6f1dbe36ee98e2b04/.mailmap).